### DISCUSSION TITLE
Project expiration

### DISCUSSION THREAD

**Discussion by c4benn** [supabase/supabase#2598] [Questions] [answered]

**Comment by inian** [comment, marked as answer]
for free tier users, we pause a project if it is inactive (no api or database calls) for a week. You can restore the project at a later point in time and there should be no data lost.

**Reply by Batalex** [reply]
Hi, 
My project was paused today (October 6th), despite being active recently :

- auth requests on October 4th
- auth requests on September 29th

Could you please confirm that an authentication with the supabase client postpones the pause?
I strongly appreciate the free tier, so thank you kindly. I have no doubt I will go with the paid plan once I go into production.

**Reply by darora** [reply]
@Batalex - auth requests _should_ postpone pauses.

We've isolated a bug that caused a small number of projects to be mislabeled as inactive; if you run into issues having your project restored, could you send the project ref to support@supabase.io and we'll sort it out?
